src/beat_number.rs

```diff
@@ -1,3 +1,4 @@
+#[derive(PartialEq, Debug)]
```

Comment:
If you’re implementing `PartialEq`, I would always implement `Eq` at the same time (assuming it is supported by your struct).